nifpga/fixedpointhelper.py

```diff
@@ -0,0 +1,59 @@
+from warnings import warn
+
+
+def bin_to_int(binary_string):
+    if binary_string == '':
+        return 0
+    else:
+        return int(binary_string, 2)
+
+
+def to_bin(value):
+    """
+    This class uses the built in bin() function to do the conversion,
+    but we remove the leading '0b' or '-0b' identifier to keep the logic
+    simplified.
+    """
+    binary_representation = bin(value)
+    if binary_representation[0] == '-':
+        return binary_representation[3:]
+    else:
+        return binary_representation[2:]
+
+
+def twos_compliment(binary_string):
+    binary_string = _flip_bits(binary_string)
```

Comment:
I'm a fan of immutable variables. It encourages the invention of new names that are descriptive of the new value. In this case, maybe ones_complement or not_binary_string. The result of _add_one_to_binary might be assigned to a new variable called negative_binary_string, or something like that.